nilearn/regions/tests/test_region_extractor.py

```diff
@@ -197,15 +226,19 @@ def test_region_extractor_fit_and_transform():
         get_data(extractor_with_mask.regions_img_)[mask_data == 0] == 0.0
     )
 
-    # smoke test to RegionExtractor with thresholding_strategy='ratio_n_voxels'
+
+def test_region_extractor_strategies(maps_and_mask):
```

Comment:
I think this could be split in two: `test_region_extractor_ratio_n_voxels_strategy` and `test_region_extractor_percentile_strategy`

Reply:
done